example/example.dart

```diff
@@ -35,7 +33,7 @@ abstract class Callbacks {
   Cat,
   Callbacks,
 ], customMocks: [
-  MockSpec<Cat>(as: #MockCatRelaxed, returnNullOnMissingStub: true),
+  MockSpec<Cat>(as: #MockCatRelaxed),
```

Comment:
We don't really show the usage of it here, but this won't make it relaxed: for `GenerateMocks` the default is to throw. If we want to show a relaxed version, we have to pass `onMissingStub: OnMissingStub.returnDefault`.

Reply:
Thanks for the call out. I specified the `OnMissingStub.returnDefault` and added an example of the resulting behavior. Let me know what you think.